File: backend/Dtos/UserRecentlyViewedItemDto.cs
namespace backend.Dtos
{
    public class UserRecentlyViewedItemDto
    {
        public int ItemId { get; set; }
        public string ItemTitle { get; set; } = string.Empty;
        public string ItemSlug { get; set; } = string.Empty;
        public string? ItemMainPhotoUrl { get; set; }
        public decimal PricePerDay { get; set; }
        public bool IsFree { get; set; }
        public bool IsAvailable { get; set; }
        public string OwnerName { get; set; } = string.Empty;
        public DateTime ViewedAt { get; set; }
    }
}

File: frontend/src/app/dtos/userRecentlyViewedItemDto.ts
import { ItemDto } from './itemDTO';

export interface UserRecentlyViewedItemResponseDto {
  item: ItemDto;
  viewedAt: string;
}

export interface UserRecentlyViewedItemDto {
  itemId: number;
  itemTitle: string;
  itemSlug: string;
  itemMainPhotoUrl: string | null;
  pricePerDay: number;
  isFree: boolean;
  isAvailable: boolean;
  ownerName: string;
  viewedAt: string;
}